modules/activity_tracker/variables.tf

```diff
@@ -152,6 +152,18 @@ variable "log_analysis_targets" {
   sensitive   = true
 }
 
+# Cloud Logs Targets
+variable "cloud_logs_targets" {
+  type = list(object({
+    instance_id   = string
+    target_region = optional(string)
+    target_name   = string
+  }))
+  default     = []
+  description = "List of Cloud logs target to be created"
+  sensitive   = true
```

Comment:
why is this marked as sensitive?

Reply:
Removed the sensitive tag as per this [discussion](https://ibm-cloudplatform.slack.com/archives/CG06FQ31P/p1725639444024319?thread_ts=1725625519.701209&cid=CG06FQ31P).